MAINTAINERS.yml

```diff
@@ -1364,6 +1364,8 @@ Release Notes:
     - dts/bindings/ethernet/
     - tests/drivers/ethernet/
     - include/zephyr/drivers/ethernet/
+    - include/zephyr/net/phy.h
+    - subsys/net/l2/ethernet/
```

Comment:
This directory is actually not related to Ethernet drivers, but to Ethernet L2, so I suggest it is omitted from this list.